.github/workflows/docker.yml

```diff
@@ -1,32 +1,38 @@
 name: Publish Docker image
 on:
-  release:
-    types: [published]
+  # release:
+  #   types: [published]
+  push:
+    branches: [ master ]
 jobs:
   push_to_registry:
-    name: Push Docker image to Docker Hub
     runs-on: ubuntu-latest
     steps:
-      - name: Check out the repo
+      - name: Checkout
         uses: actions/checkout@v2
-      - name: Push to Docker Hub
-        uses: docker/build-push-action@v1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Build
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: prismarinejs/flying-squid
-          path: deploy
-          registry: docker.io
-          dockerfile: deploy/Dockerfile
-          tag_with_ref: true
-
-      - name: Push to GitHub Packages
-        uses: docker/build-push-action@v1
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1 
         with:
+          registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-          repository: prismarinejs/flying-squid/flying-squid
-          path: deploy
-          registry: docker.pkg.github.com
-          dockerfile: deploy/Dockerfile
-          tag_with_ref: true
\ No newline at end of file
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
```

Comment:
I think the path (context) was "." and not deploy i.e. when I build locally I do `docker build -t prismarinejs/flying-squid -f deploy/Dockerfile .` (from root folder). I updated to the v2 action too which is maybe a little bit less verbose

see example [here](https://github.com/louis030195/niwrad/blob/0c3f348bcf31457d90d08c39b3b35a3c64940f7c/.github/workflows/main.yml#L38)